helm_deploy/django-app/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "django-app.fullname" . }}
+  labels:
+    app: {{ template "django-app.name" . }}
+    chart: {{ template "django-app.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "django-app.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "django-app.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: SECRET_KEY
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "django-app.fullname" . }}
+                key: secretKey
+          - name: SERVER_NAME
+            value: {{ default "" .Values.ingress.hosts | quote }}
+          - name: POSTGRES_USER
```

Comment:
I'd say that the postgres user is a secret too, so this should be added to the secrets object and referenced there.